internal/http/activity.go

```diff
@@ -44,7 +42,7 @@ func getActivityHandler(w http.ResponseWriter, r *http.Request) {
 
 		break
 	default:
-		error.Error(fmt.Errorf("action %s not found", vars["action"]))
+		error.Error(fmt.Errorf("action %server not found", vars["action"]))
```

Comment:
I'm guessing you searched and replaced? This one should stay as `%s`.

Reply:
yep yep! just changed back